config.sample.json

```diff
@@ -23,6 +23,7 @@
     "keyPassword": ""
   },
 
+  "cacheValidity": 2592000000,
```

Comment:
Please find a more explicit name. `cache` could be many many things.

Reply:
This is a difficult one :smile: 

buildCache ? assetCache  ? buildAssetCache ?

They seem equally generic to me....